libbeat/template/config.go

```diff
@@ -1,12 +1,13 @@
 package template
 
 type TemplateConfig struct {
-	Enabled      bool                   `config:"enabled"`
-	Name         string                 `config:"name"`
-	Fields       string                 `config:"fields"`
-	Overwrite    bool                   `config:"overwrite"`
-	OutputToFile string                 `config:"output_to_file"`
-	Settings     map[string]interface{} `config:"settings"`
+	Enabled        bool                   `config:"enabled"`
+	Name           string                 `config:"name"`
+	Fields         string                 `config:"fields"`
+	Overwrite      bool                   `config:"overwrite"`
+	OutputToFile   string                 `config:"output_to_file"`
+	Settings       map[string]interface{} `config:"settings"`
+	SourceSettings map[string]interface{} `config:"_source"`
```

Comment:
At first I was also sceptical about using _source but I think for ES users it must be quite intuitive to use it like this.